VideoLocker/src/main/java/org/edx/mobile/model/VerticalModel.java

```diff
@@ -48,4 +50,25 @@ public String getUnitUrl() {
     public void setUnitUrl(String unitUrl) {
         this.unitUrl = unitUrl;
     }
+
+    @Override
+    public int getVideoCount(){
+        int count = 0;
+        for( IUnit unit :units ){
+            //TODO - we may need to create enum type for categories
+           if ("video".equals(unit.getCategory())  ){
```

Comment:
@aleffert the data model will be changed to match the new API.  i plan to do it as next assessment story